src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-anchor.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AlloyEditor from 'alloyeditor';
+
+export default class EzBtnAnchor extends Component {
+    constructor(props) {
+        super(props);
+
+        this.getStateClasses = AlloyEditor.ButtonStateClasses.getStateClasses;
+    }
+
+    static get key() {
+        return 'ezanchor';
+    }
+
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        if (this.props.renderExclusive) {
+            return <AlloyEditor.EzBtnAnchorEdit {...this.props} />;
+        }
+
+        const cssClass = 'ae-button ez-btn-ae--anchor ez-btn-ae ' + this.getStateClasses();
```

Comment:
Nitpick: we usually use `` `${}` `` to concatenate strings. Do we have any in-house rule on this? There is eslint rule for this: https://eslint.org/docs/rules/prefer-template.

Reply:
Yes, most of the button in AE has the convention that I proposed. We should unify this in 3.0.